database/user.go

```diff
@@ -70,14 +68,21 @@ type (
 		ID              primitive.ObjectID `bson:"_id,omitempty" json:"-"`
 		Sub             string             `bson:"sub" json:"sub"`
 		Tier            int                `bson:"tier" json:"tier"`
-		StorageUsed     int64              `bson:"storage_used" json:"storageUsed"`
 		SubscribedUntil time.Time          `bson:"subscribed_until" json:"subscribedUntil"`
 	}
-	// UserDetails builds on top of User and provides additional information,
-	// typically more expensive to fetch than a single DB query.
-	UserDetails struct {
-		User
-		BandwidthUsed int64 `bson:"-" json:"bandwidthUsed"`
+	// UserStats contains statistical information about the user.
+	UserStats struct {
+		StorageUsed        int64 `bson:"-" json:"storageUsed"`
```

Comment:
in siad we have lowercase characters for all json fields, I kind of like that, should we be consistent here?

Reply:
The name convention of the JSON side of things is requested by Karol in order to be in line with what's usually used in JS. Personally, I don't mind either way. If you have a strong opinion on this one you can ping @kwypchlo and you can hash it out. I'm happy to follow any convention.